Cargo.toml

```diff
@@ -301,3 +303,11 @@ inherits = "release"
 lto = "off"
 opt-level = 0
 strip = true
+
+
+[patch.crates-io]
+jsonrpsee = { git = "https://github.com/paritytech/jsonrpsee", rev = "ffd5020e9c2940e1d5cfb49900b4a17518f24ec3" }
+jsonrpsee-http-client = { git = "https://github.com/paritytech/jsonrpsee", rev = "ffd5020e9c2940e1d5cfb49900b4a17518f24ec3" }
+jsonrpsee-types = { git = "https://github.com/paritytech/jsonrpsee", rev = "ffd5020e9c2940e1d5cfb49900b4a17518f24ec3" }
+jsonrpsee-core = { git = "https://github.com/paritytech/jsonrpsee", rev = "ffd5020e9c2940e1d5cfb49900b4a17518f24ec3" }
+jsonrpsee-server = { git = "https://github.com/paritytech/jsonrpsee", rev = "ffd5020e9c2940e1d5cfb49900b4a17518f24ec3" }
```

Comment:
Why do we need these patches?
This is exactly version 0.26,
see the pic below from https://github.com/paritytech/jsonrpsee/commit/ffd5020e9c2940e1d5cfb49900b4a17518f24ec3

<img width="1636" height="892" alt="Image" src="https://github.com/user-attachments/assets/49656d64-e117-4c9d-90cf-9dd62ee8059e" />